packages/editor/src/components/browser/Browser.css

```diff
@@ -67,6 +69,16 @@
   width: unset;
 }
 
+.dialog-content .browser-helptext p.helperTitle {
+  margin: 0 0 var(--size-2) 0;
+  font-weight: bold;
+  font-size: 12;
+}
+
+.dialog-content .browser-helptext .cmsValue {
+  margin-top: var(--size-1);
+}
+
```

Comment:
didn't I told you why it is not a good idea to use margins? If the p has default margins then set those to 0 or just don't use 'p'
Better add a display flex on the parent (.browser-helptext) with a gap. And the font-size is not needed on the title I think, as you already define it on the parent.

If you do so you don't need to do any stylings and you also don't need to give the value an own class, so you can simplify the html code a lot too. Se below.